numpy/_core/strings.py

```diff
@@ -31,16 +32,272 @@
 
 
 __all__ = [
+    # UFuncs
     "equal", "not_equal", "less", "less_equal", "greater", "greater_equal",
     "add", "isalpha", "isdigit", "isspace", "isdecimal", "isnumeric",
     "str_len", "find", "rfind", "count", "startswith", "endswith",
-    "lstrip", "rstrip", "strip", "replace"
+    "lstrip", "rstrip", "strip", "replace",
+
+    # _vec_string
```

Comment:
Maybe add a line to this comment saying the plan is to move all these to ufuncs.